package test.trystmt;

class TryStmtTest2 {

    void m() {
        int a = 0;
        try {
            a = mm();
        } catch (Exception e) {
            a = 2;
        } finally {
            a = 5;
        }
        a = 3;
    }

    int mm () throws Exception {
        throw new Exception("");
    }

}
